### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by oniram22**
See issue #39 for background information on what's being submitted here.  Thanks.

### CODE REVIEW

File: samples/__init__.py [hunk 1]
```diff
@@ -0,0 +1,836 @@
...
+            return ""
+
+    def add_node_using_snmp_v3(self, node_name, ip_address, snmpv3_username, snmpv3_priv_method, snmpv3_priv_pwd,
+                               snmpv3_auth_method, snmpv3_auth_pwd):
+        """ Creates a new node using the supplied name an ip address.  Configure with our standard SNMPv3 credentials.
+            Once created, attached all of the standard Cisco pollers.
+
+            Args:
+                node_name(string): A node name to be used for the newly created node object.
+                ip_address(string): The ip address that is associated with the supplied node name.
```
Comment (danjagnow) [inline samples/__init__.py]: I'd capitalize "IP" in the description.

File: samples/__init__.py [hunk 2]
```diff
@@ -0,0 +1,836 @@
...
+
+        self.logger.info("get_node_id - node_id query results: %s.", node_id)
+
+        if node_id['results']:
+            return node_id['results'][0]['NodeID']
+        else:
+            return ""
+
+    def get_node_uri(self, node_name):
+        """ Returns the NodeURI for the given NodeName.  Uses a Swis query to the Solarwinds database to retrieve this
```
Comment (danjagnow) [inline samples/__init__.py]: "Swis" should be "SWIS" in the comments.

File: samples/__init__.py [hunk 3]
```diff
@@ -0,0 +1,836 @@
...
+                                   group_name=group_name)
+        self.logger.info("get_group_id - group_id query results: %s", group_id)
+
+        if group_id['results']:
+            return group_id['results'][0]['ContainerID']
+        else:
+            return ""
+
+    def get_group_uri(self, group_name):
+        """ Returns the ContainerUrifor the given Group Name.  Uses a Swis query to the Solarwinds database to retrieve this
```
Comment (danjagnow) [inline samples/__init__.py]: space before "for"

**Comment by oniram22** [comment]
I agree with all the recommended edit’s.  I corrected them and submitted a new pull request with the changes.

Michael J. Marino
Assistant Director of Network Services
Morgan, Lewis & Bockius LLP
1701 Market Street | Philadelphia, PA 19103-2921
Direct: +1.215.963.4658<tel:+1.215.963.4658> | Main: +1.215.963.5000<tel:+1.215.963.5000> | Fax: +1.215.963.5001<tel:+1.215.963.5001>
michael.marino@morganlewis.com<mailto:michael.marino@morganlewis.com> | www.morganlewis.com<http://www.morganlewis.com/>

**Pull request merged**